src/main/java/cam72cam/mod/world/World.java

```diff
@@ -654,6 +656,13 @@ public List<ItemStack> getDroppedItems(IBoundingBox bb) {
         return items.stream().map((ItemEntity::getItem)).map(ItemStack::new).collect(Collectors.toList());
     }
 
+    /** Get dropped items within the given area in entity form*/
+    public List<cam72cam.mod.entity.ItemEntity> getDroppedItemWithinBB(IBoundingBox bb) {
```

Comment:
Nit: getDroppedItemWithinBB -> getDroppedItemEntities to follow the naming convention.